images/ceph/Makefile

```diff
@@ -17,7 +17,7 @@ include ../image.mk
 # ====================================================================================
 # Image Build Options
 
-CEPH_VERSION = v14.2.4-20190917
+CEPH_VERSION = v14.2.4-20191204
```

Comment:
The arm64 tag doesn't exist for this. Looks like the last arm64 tag is `ceph/ceph-arm64:v14.2.4-20191112`

Reply:
Yes, I asked @dsavineau to look into this. We just need to fire a new build.